lib/dash/dash_parser.js

```diff
@@ -648,6 +648,11 @@ shaka.dash.DashParser = class {
     // best latency
     if (this.lowLatencyMode_) {
       minBufferTime = 0;
+      const defaultPresentationDelay =
```

Comment:
Since we just set `minBufferTime` to 0, and `this.config_.defaultPresentationDelay` is non-nullable, this isn't necessary. You could just do:

```js
const presentationDelay = suggestedPresentationDelay != null ?
          suggestedPresentationDelay : this.config_.defaultPresentationDelay;
```

Reply:
Done!